src/facade/dragonfly_connection.h

```diff
@@ -19,6 +19,10 @@
 typedef struct ssl_ctx_st SSL_CTX;
 typedef struct mi_heap_s mi_heap_t;
 
+namespace dfly {
+class ConnectionState;
```

Comment:
this creates a circular linking dependency. 
The way the code is designed, "server" is dependent on "core" and "facade". "facade" can not be dependent on the server code. I bet the "ok_backend" target does not build anymore.

Reply:
So I need to make sure that I do not use server from inside facade, I will make the required changes for that